LocalyticsXamarin/Android/FCMClient.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Android.App;
+using Firebase.Iid;
+using Android.Util;
+
+namespace LocalyticsSample.Android
+{
+    [Service]
+    [IntentFilter(new[] { "com.google.firebase.INSTANCE_ID_EVENT" })]
+    public class MyFirebaseIIDService : FirebaseInstanceIdService
```

Comment:
This is the Class that we should be gtting rid of now that we are on a newer version of Firebase.  As of Firebase 16 or 17 (Forget which exactly at this point) the ability to receive a push token has been moved to the `FirebaseMessagingService` in the method `onNewToken`

Reply:
I lifted this from the Microsoft docs. https://docs.microsoft.com/en-us/xamarin/android/data-cloud/google-messaging/remote-notifications-with-fcm?tabs=macos

But yeah, seems out of date at this point. We don't need this for the app and it was just a part of the tinkering code to try and force the sample app to connect to firebase, so I'll just remove it.